Google Books.js

```diff
@@ -116,17 +116,22 @@ function parseXML(text) {
 	}
 	
 	var ISBN;
-	const ISBN10Re = /(ISBN:)(\w{10})$/;
-	const ISBN13Re = /(ISBN:)(\w{13})$/;
+	const ISBN10Re = /(?:ISBN:)(\w{10})$/;
+	const ISBN13Re = /(?:ISBN:)(\w{13})$/;
+	const booksIDRe = /^(\w{12})$/;
 	var identifiers = ZU.xpath(xml, "dc:identifier", ns);
 	for (var i in identifiers) {
 		var ISBN10Match = ISBN10Re.exec(identifiers[i].textContent);
 		var ISBN13Match = ISBN13Re.exec(identifiers[i].textContent);
+		var booksIDMatch = booksIDRe.exec(identifiers[i].textContent);
 		if (ISBN10Match != null) {
-			ISBN = ISBN10Match[2];
+			ISBN = ISBN10Match[1];
 		}
 		if (ISBN13Match != null) {
-			ISBN = ISBN13Match[2];
+			ISBN = ISBN13Match[1];
+		}
+		if (booksIDMatch != null) {
+			newItem.url = 'https://books.google.com/books?id=' + booksIDMatch[1];
```

Comment:
Hm.. Okay, I guess you are right. This is similar to not to save the link to amazon in the url field, just because the metadata where taken from there.

On the other hand there is a Wikidata property [P675](https://www.wikidata.org/wiki/Property:P675) for the Google-Books-ID, which is in particularly used in giving references to statements.

Should we add the Google-Books-ID in the `extra` field, i.e.
```js
newItem.extra = "Google-Books-ID: " + booksIDMatch[1];
```
? Then @mvolz can take it from there for the Citoid service?